### ISSUE TITLE
NullPointerException on calling analyze() if no record exists

### ISSUE THREAD

**Issue by inhaledesign** [T-Spoon/Benchit#4]
If I call Benchit.analyze("test") when no records for "test" have been recorded, I'll get this exception:

`java.lang.NullPointerException: Attempt to invoke virtual method 'com.tspoon.benchit.Result com.tspoon.benchit.Benchmark.result()' on a null object reference`

I know it's kindof minor and I could catch the exception myself, but it would be preferable if an empty result could be returned and .log() would print out a message indicating there were no records.

**Comment by T-Spoon** [comment]
Makes sense - should be an easy fix.  I'll see if I can take care of this soon!
